src/components/CardListItem.js

```diff
@@ -14,6 +14,7 @@ export class CardListItem extends React.PureComponent {
     const { navigation, horizontal, item } = this.props;
     const { routeName, params, image, category, name } = item;
 
+    // TODO accessibility logic needs to be implemented
```

Comment:
currently it is read strange with "in progress {category} {title}" without pause.
maybe we need a accessible "({category})  {title} (Taste)", right? to have nice pause while reading between category, title and taste